modules/kubectl-wrapper/README.md

```diff
@@ -0,0 +1,38 @@
+# kubectl wrapper
+
+This submodule aims to make interactions with GKE clusters using kubectl easier by utilizing the gcloud module and kubectl_wrapper script.
```

Comment:
Let's link to the example and add a bit more explanation of why this is useful.

Reply:
Added link to example, the use case and basic usage. Let me know if you think anything else maybe useful.